### ISSUE TITLE
SQL Editor scroll automatic scroll when typing -> unable to see what we are typing when there are a lot of queries on the left

### ISSUE THREAD

**Issue by florianoger** [supabase/supabase#21291]
# Bug report

- [X ] I confirm this is a bug with Supabase, not with my own application. 
- [ X] I confirm I have searched the [Docs](https://docs.supabase.com), GitHub [Discussions](https://github.com/supabase/supabase/discussions), and [Discord](https://discord.supabase.com). 

## Describe the bug
On SQL Editor, when I have a lot of saved queries on the left bar, and that my query is in the list, but not seen on screen when there is no scroll, there will be a focus on the current query, and scroll where it is saved. This will scroll the entire screen, and I am unable to see what I am typing.in my query. 

## To Reproduce

1. Go to SQL Editor
2. Have a queries list bigger than your screen can display
3. Select a query below the screen limit
4. Type something in the query

## Expected behavior

No scroll

## Screenshots

Note that in the video, I never scroll down, this is done automatically.

https://github.com/supabase/supabase/assets/125586990/5010c911-81af-4179-ab60-4946e35755c3

## System information

- OS: macOS Sonoma 14.1.1
- Browser: safari

**Comment by Mohammed-Yasin-Mulla** [comment]
@florianoger  I can't replicate the issue, could you please explain how to replicate it or maybe it's a Safari bug? 
https://github.com/supabase/supabase/assets/55092280/04b1561a-f0ad-4cb8-a3e2-1e193cefa0fc

**Comment by florianoger** [comment]
@Mohammed-Yasin-Mulla Hi, yes, I tried on Chrome and it is working exactly like you. I figured out that the focus was made if you stop typing in the editor, after 1-2 seconds (when you are in your list at the top and your current query is bellow). I don't know why the scroll bar in the list of queries on Safari does not exist. This was working well a few weeks ago, maybe introduced since this release : https://supabase.com/changelog#sql-editor-bulk-deletes

In the video, the left part is Chrome, and right part is Safari. We can see that the entire screen scrolls on Safari, but not on Chrome. So the focus is not a problem on Chrome, but makes problems on Safari.

https://github.com/supabase/supabase/assets/125586990/4ab7a89b-8e0f-4394-b018-0915a60fb59d
